OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -42,14 +42,13 @@ using namespace std;
 /**
  * Default constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame* frame)
 {
```

Comment:
Actually, I'm not sure anymore if it's outside the scope of the PR. I think it's fine we remove the logic that checks for null.